src/boards/include/boards/adafruit_fruit_jam.h

```diff
@@ -28,6 +28,67 @@
 // This means RP2350B.
 #define PICO_RP2350A 0
 
+// --- BOARD SPECIFIC ---
+// On JST PH connector.
+#define ADAFRUIT_FRUIT_JAM_A0_PIN 40
+
+#define ADAFRUIT_FRUIT_JAM_A1_PIN 41
+#define ADAFRUIT_FRUIT_JAM_A2_PIN 42
+#define ADAFRUIT_FRUIT_JAM_A3_PIN 43
+#define ADAFRUIT_FRUIT_JAM_A4_PIN 44
+#define ADAFRUIT_FRUIT_JAM_A5_PIN 45
+
+#define ADAFRUIT_FRUIT_JAM_D6_PIN 6
+#define ADAFRUIT_FRUIT_JAM_D7_PIN 7
+#define ADAFRUIT_FRUIT_JAM_D8_PIN 8
+#define ADAFRUIT_FRUIT_JAM_D9_PIN 9
+#define ADAFRUIT_FRUIT_JAM_D10_PIN 10
+
+#define ADAFRUIT_FRUIT_JAM_BOOT_BUTTON_PIN 0
+
+#define ADAFRUIT_FRUIT_JAM_BUTTON1_PIN 4
+#define ADAFRUIT_FRUIT_JAM_BUTTON2_PIN 5
+
+#define ADAFRUIT_FRUIT_JAM_ESP_CS_PIN 46
+
+#define ADAFRUIT_FRUIT_JAM_CKN_PIN 12
+#define ADAFRUIT_FRUIT_JAM_CKP_PIN 13
+#define ADAFRUIT_FRUIT_JAM_D0N_PIN 14
+#define ADAFRUIT_FRUIT_JAM_D0P_PIN 15
+#define ADAFRUIT_FRUIT_JAM_D1N_PIN 16
+#define ADAFRUIT_FRUIT_JAM_D1P_PIN 17
+#define ADAFRUIT_FRUIT_JAM_D2N_PIN 18
+#define ADAFRUIT_FRUIT_JAM_D2P_PIN 19
```

Comment:
Like you've done with some of your other pin-defines, I wonder if it's worth inserting an additional word between e.g. `JAM` and `CKN` ?

Reply:
These are the DVI pin names, for doing DVI over HSTX. We haven't been including `DVI` on the CircuitPython boards with these pins, but it wouldn't hurt. I'll do that.